src/net_packet.c

```diff
@@ -82,7 +82,11 @@ static void send_mtu_probe_handler(event_loop_t *loop, void *data) {
 		n->status.udp_confirmed = false;
 		n->mtuprobes = 1;
 		n->minmtu = 0;
-		n->maxmtu = MTU;
+		n->maxmtu = sptps_maxmtu(&n->sptps);
+
+		// reduce mtu a bit as well
```

Comment:
not necessary, but when increased along with minmtu it's likely it got too large as well so while the detection is reset for the mean time till probes become 30 again I thought to use a more reasonable value with chance to work.
MTU usually doesn't change much but by some slight header overhead in some networks.
But I'm ok to remove it along with changes in line 201-206.